network/load-balancer/how-to/add-certificate.mdx

```diff
@@ -0,0 +1,162 @@
+    - Select **Import Certificate** under type.
+    - Paste the full PEM-formatted certificate into the box provided. Include the entire certificate chain with public key, private key, and (optionally) Certificate Authorities. You may have received this information either in a combined file or in several separate files from your CA. Contact them for more details on how to combine the information if you received the chain divided into multiple files.
+
+6. Click **Create SSL certificate**.
+
+    You are returned to the **SSL certificates** tab where your newly-created certificate now appears. You must now attach the certificate to your frontend:
+
+    <Lightbox src="scaleway-create-self-certificate.webp" />	
+
+7. Click the **Frontends** tab.
+
+8. Click the three dot menu next to the frontend you want to add the certificate to, then click **Edit**
+
+    <Lightbox src="scaleway-edit-frontend.webp" alt="" />
+
+9. Select the certificate you just created from the **SSL Certificate(s)** dropdown, then click **Edit Frontend**.
+
+Your certificate is added to your Load Balancer's frontend.
+
+## How to create a self-signed certificate
+
+If you are using the [import certificate](#how-to-import-a-certificate) option, you may wish to generate your own self-signed certificate to import to your Load Balancer. These certificates are not signed by any publically-trusted Certificate Authority, but may be suitable for development ad testing purposes, or internal use. A self-signed certificate can be issued easily on any computer using the `openssl` tool. Follow these steps to generate a self-signed certificate:
+
+1. From your computer, open up a command line terminal and run the following command to generate a private key file and a CSR file:
+	```
+	openssl req -new -newkey rsa:4096 -nodes -keyout private_key.txt -out csr.txt -subj "/C=FR/ST=Ile-de-France/L=Paris/O=MyInternetCompanyLTD/CN=www.example.com"
+	```
+	The following subcommands are used with the `openssl` base command:
+	- `req`: This subcommand specifies the use of the X.509 certificate signing request (CSR) management.
+	- `newkey rsa: 4096`:  This subcommand specifies the creation of a new key and certificate at the same time using a 4096 bit long RSA key.
+	- `nodes`: This option tells OpenSSL to skip the securisation of the certificate using a passphrase.
+	- `keyout`: This subcommand defines the location and file name of the private key file
+	- `out`: This specifies the path and file name of the generated certificate request.
+	- `sub`: The subject for the certficate. Modify the values of these subcommand to your requirements. The **Common Name** (CN) represents the domain name you are issuing the certificate for.
+
+2. Generate a file containing all **Subject Alternative Names** (SAN) for the certificate. They include websites, IP addresses, common names, etc. They are protected by a single SSL Certificate. 
```

Comment:
Maybe we can combine this step with step 3 because it's a bit confusing in this way.

Reply:
done